tests/ImageSharp.Tests/Image/ImageFramesCollectionTests.cs

```diff
@@ -15,6 +15,9 @@ public class ImageFramesCollectionTests : IDisposable
 
         public ImageFramesCollectionTests()
         {
+            // Needed to get English exception messages, which are checked in several tests.
+            System.Threading.Thread.CurrentThread.CurrentUICulture = new System.Globalization.CultureInfo("en-US");
```

Comment:
Good fix, thanks! 

TBH though we should be testing the content of the error message, rather simply that the error is thrown.

Reply:
You mean should not? In that case I agree! Isn't my code though, so I didn't want to mess with it too much.

Reply:
Yeah should not. The content is unimportant

Reply:
We can simply drop those assertions IMO.